sefaria/model/schema.py

```diff
@@ -1374,11 +1374,11 @@ def version_address(self):
         """
         return self.address()[1:]
 
-    def ref(self):
+    def ref(self, force_update=False):
         from . import text
         d = {
             "index": self.index,
-            "book": self.full_title("en"),
+            "book": self.full_title("en", force_update=force_update),
```

Comment:
The change here is fine in the current context, that being said - the proper thing to do here would be to address the underlying cache issue.